app/lib/onboarding/v2/onboarding_categories_production.yml

```diff
@@ -36,6 +36,9 @@
     - account: '@jquillin@fashionsocial.host'
       type: 'account'
       summary: Glass Magazine Fmr Fashion editor.
+    - hashtag: "#catsofmastodon"
+      bio: cats!
+      summary: "everyone loves cats!"
```

Comment:
Capitalize "Everyone"

Reply:
removed this one entirely, I didn't like my captions.